packages/core/lib/commands/test/run.js

```diff
@@ -30,6 +30,30 @@ const parseCommandLineFlags = options => {
   }
 };
 
+// Sanitize ganache options specified in the config
+function sanitizeGanacheOptions(ganacheOptions) {
+  let network_id = ganacheOptions.network_id;
+  let ganacheOptionsChanged;
+
+  // Use default network_id if "*" is defined in config
+  if (network_id === "*") {
+    network_id = 4447;
```

Comment:
network_id is returned as 4447 (default ganache network_id) when "*" is specified in the truffle-config.js "test" network. I skipped the Date.now() for now to keep the code in sync with `truffle develop`.

Reply:
Can you replace the magic number with an appropriately commented constant?

Reply:
> Can you replace the magic number with an appropriately commented constant?

Or should I go with Date.now() as you suggested?

Reply:
> Can you replace the magic number with an appropriately commented constant?

Sure, @cds-amal . Thanks for the comment. :)